draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -62,6 +62,10 @@ informative:
       -
         ins: H. Krawczyk
         org: IBM Research, NY, USA
+  XXXX:
+    title:  "To Appear"
+    target: https://eprint.iacr.org/2022/XXXX
+    date: March, 2022
```

Comment:
The paper should be live ahead of the next IETF meeting draft upload deadline

Reply:
Huzzah!